client/fns_test.go

```diff
@@ -34,3 +34,50 @@ func TestVsockIdPort(t *testing.T) {
 		}
 	}
 }
+
+func TestParseBinds(t *testing.T) {
+	for _, tt := range []struct {
+		namespace string
+		fstab     string
```

Comment:
minor, but it might be easier to read the tests if fstab was last?

Reply:
yes. It would. Will fix.